tests/testthat/_snaps/bump_version.md

```diff
@@ -9,45 +9,23 @@
       -- Updating NEWS --
       
       > Adding new entries to 'NEWS.md'.
-      
```

Comment:
Why has this output changed?

Reply:
I'd like to revisit this once #152 is merged as I think the variants will help see this. From quick "manual" tests, nothing is broken, but better to rebase etc.